packages/shared/ReactFeatureFlags.js

```diff
@@ -37,6 +37,9 @@ export const replayFailedUnitOfWorkWithInvokeGuardedCallback = __DEV__;
 // Warn about deprecated, async-unsafe lifecycles; relates to RFC #6:
 export const warnAboutDeprecatedLifecycles = false;
 
+// Controls enabling of experimental multi-callback support in 'schedule'
+export const scheduleModuleSupportsMultipleCallbacks = false;
```

Comment:
Why is this a feature flag? I don't think there's any valid use of the schedule module unless it supports multiple callbacks. Except for its use by React, but in that case it's only used for async mode, which is already behind its own feature flag. I don't think we need this one.